drivers/SmartThings/matter-appliance/profiles/dishwasher-tn.yml

```diff
@@ -6,10 +6,6 @@ components:
     version: 1
   - id: temperatureSetpoint
     version: 1
-    config:
```

Comment:
@HunsupJung can we not use embedded config values when we have metadata? Do you know if we are still able to set these limits via a function call in the driver? For example, would something like `capabilities.thermostatHeatingSetpoint.heatingSetpointRange()` still work if we are using a profile with metadata?

Reply:
> Do you know if we are still able to set these limits via a function call in the driver? For example, would something like capabilities.thermostatHeatingSetpoint.heatingSetpointRange() still work if we are using a profile with metadata?

It doesn't work. If `embedded config` is set, `metadata` setting does not work. and the `temperature range` set in the `embedded config` can be set in `device configuration`. so it is removed from profile.